### ISSUE TITLE
[Change Request] Change that after reboot also for battery devices the latest values are shown

### ISSUE THREAD

**Issue by djanssen1** [athombv/homey#1598]
**What is your Homey version (Settings → System)?**

1.3.0 RC11

After a reboot, values of battery devices that are older than 30 minutes are not showed. 
Please make this time configurable, change it to 24 hours, or just display the values as you would be able to detect it is a reboot

**Comment by corvdniet** [comment]
I would call it a bug. In 1.3.0 RC9 showing the battery status after update worked (again) for my Fibaro sensor. Now in 1.3.0 RC11 the "old bug" is back.

**Comment by djanssen1** [comment]
Same here. Also battery status is gone. On slack caseda told that if a value is more than 30 minutes old it will not be saved. So not sure if they is also applicable for battery level.

**Comment by jaimevisser** [comment]
Battery values should never be considered "expired". Last known value is good enough in almost all cases.

**Comment by caseda** [comment]
Yeah, and then have people complaining that the sensor doesn't work anymore even though the battery level said something in between 50 and 100% :smile:
Just go complain with fibaro that they don't really measure the battery level and send it with every request.
